Xamarin.Forms.Core/Color.cs

```diff
@@ -22,11 +24,14 @@ public static Color Default
 			get { return new Color(-1d, -1d, -1d, -1d, Mode.Default); }
 		}
 
-		internal bool IsDefault
+		[EditorBrowsable(EditorBrowsableState.Never)]
+		public bool IsDefault
```

Comment:
I'd rather this PR not change the public surface area exposed to non-platform implementors. Let's leave those for another PR.